cli/check-unused-langterms.js

```diff
@@ -0,0 +1,77 @@
+import { dirname, join, resolve } from 'path';
```

Comment:
File is copy of rubrics usage here: https://github.com/Brightspace/d2l-rubric/blob/main/cli/check-unused-langterms.js

Reply:
How difficult do you think it would be to add this to the existing `mfv` library? I'm sure @bearfriend would be interested...

Reply:
Not entirely sure how hard it would be, not super familiar with the `mfv` library. Could potentially be a good inspiration sprint idea?